app/controllers/catalog_controller.rb

```diff
@@ -57,21 +57,20 @@ class CatalogController < ApplicationController
     # exploded_tag_ssim indexes all tag prefixes (see IdentityMetadataDS#to_solr for a more exact
     # description), whereas tag_ssim only indexes whole tags.  we want to facet on exploded_tag_ssim
     # to get the hierarchy.
-    config.add_facet_field 'exploded_tag_ssim',                 label: 'Tag',                 limit: 9999,
```

Comment:
This is mostly just spacing changes. You may want to "hide whitespace" when reviewing.